hs/src/Reach/AST/DL.hs

```diff
@@ -104,7 +105,7 @@ data DLSStmt
   | DLS_While
       { dls_w_at :: SrcLoc
       , dls_w_asn :: DLAssignment
-      , dls_w_inv :: DLSBlock
+      , dls_w_inv :: [(DLSBlock, Maybe ByteString)]
```

Comment:
You should make a type for this. A question I have now is whether any bindings are shared (I don't think they need to be, but I'm curious)

Reply:
OK & the bindings aren't shared